### ISSUE TITLE
Incorrect test values in accumulator unit test

### ISSUE THREAD

**Issue by jsdelfino** [cloudfoundry-incubator/cf-abacus#115]
The test values in the accumulator unit test seem to be incorrect. This test breaks the whole build so I've commented out the incorrect section of the test for now, but that test needs to be fixed as soon as possible.

**Comment by jsdelfino** [comment]
That test should really use:
```
 clock = sinon.useFakeTimers(Date.now(),
  'Date', 'setTimeout', 'clearTimeout', 'setInterval', 'clearInterval');
```
to control the passing of time during the execution of the test and make it deterministic.

**Comment by jsdelfino** [comment]
That would help simplify the test logic simpler as IMO it's way too complicated right now.

**Comment by jsdelfino** [comment]
Actually fixed in 6a92c41.
